pkg/template/template.go

```
@@ -40,20 +41,49 @@ func (p *Processor) Process(template *api.Template) (*configapi.Config, fielderr
 	}
 
 	for i, item := range template.Objects {
+		if obj, ok := item.(*runtime.Unknown); ok {
```

`DecodeList` ahead of time, ignore the errors, and skip over `runtime.Unknown`?